<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddObjectDialog</class>
 <widget class="QDialog" name="AddObjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>251</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="noradEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>SIC</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Norad</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QListWidget" name="setsListWidget">
      <property name="selectionMode">
       <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>ILRS ID</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="ilrsEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="aliasEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>COSPAR</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="sicEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Alias</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="cosparEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="nameEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Observation sets</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>261</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Altitude</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>NPI</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Is Debris</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="lrrCombo">
      <item>
       <property name="text">
        <string>Unknown</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>True</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>False</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="debrisCombo">
      <item>
       <property name="text">
        <string>Unknown</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>True</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>False</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Bin Size</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Rcs</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Has LRR</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="altitudeEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="rcsEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="npiEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="bsEdit"/>
    </item>
    <item row="6" column="1">
     <widget class="QListWidget" name="groupsListWidget">
      <property name="selectionMode">
       <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Groups</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>30</y>
     <width>325</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="6" column="1">
     <widget class="QPushButton" name="browseImageBtn">
      <property name="text">
       <string>Search Image</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="comEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="classEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Classification</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPlainTextEdit" name="commentsEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="configEdit"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Comments</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>CPF Provider</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="cpfEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Track HP</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="highPowerCheck">
      <property name="text">
       <string>High Power</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>CoM</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="imagePathEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="incEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Configurations</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Inc</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="selectDBImageBtn">
      <property name="text">
       <string>Select existing Image From BD</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>80</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Object</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>120</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>noradEdit</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>aliasEdit</tabstop>
  <tabstop>cosparEdit</tabstop>
  <tabstop>ilrsEdit</tabstop>
  <tabstop>sicEdit</tabstop>
  <tabstop>lrrCombo</tabstop>
  <tabstop>debrisCombo</tabstop>
  <tabstop>altitudeEdit</tabstop>
  <tabstop>rcsEdit</tabstop>
  <tabstop>npiEdit</tabstop>
  <tabstop>bsEdit</tabstop>
  <tabstop>incEdit</tabstop>
  <tabstop>comEdit</tabstop>
  <tabstop>cpfEdit</tabstop>
  <tabstop>highPowerCheck</tabstop>
  <tabstop>configEdit</tabstop>
  <tabstop>browseImageBtn</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>setsListWidget</tabstop>
  <tabstop>imagePathEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
